management/Makefile

```diff
@@ -1,132 +1,76 @@
-# NAME, LOCATION, PROJECT in Kptfile are set by calling kpt cfg set
-#
-# NAME: The cluster name of the management cluster. Warning, it should be different
-#   from your Kubeflow cluster.
-# LOCATION: Location of the management cluster. You can choose either regional or zonal.
-# PROJECT: Google Cloud project where this management cluster is created in.
-#
-# You can set these values by
-# ```
-# kpt cfg set ./instance name NAME
-# kpt cfg set ./instance location LOCATION
-# kpt cfg set ./instance gcloud.core.project PROJECT
-# ```
-# The values you set will be stored in ./instance/Kptfile, so they will be preserved.
+# NAME, LOCATION, PROJECT in Kptfile are set in ./set-values.sh
+# Please edit ./set-values.sh before calling this Makefile.
 
 NAME=$(shell yq r ./instance/Kptfile 'openAPI.definitions."io.k8s.cli.setters.name".x-k8s-cli.setter.value')
 LOCATION=$(shell yq r ./instance/Kptfile 'openAPI.definitions."io.k8s.cli.setters.location".x-k8s-cli.setter.value')
 PROJECT=$(shell yq r ./instance/Kptfile 'openAPI.definitions."io.k8s.cli.setters.gcloud.core.project".x-k8s-cli.setter.value')
 
-MGMTCTXT=$(NAME)
+export MGMTCTXT=$(NAME)
 
 # The URL you want to fetch upstream package from
-MANIFESTS_URL=https://github.com/kubeflow/gcp-blueprints.git/packages/management@master
-
-# For development, you can update the following line to point to your own branch.
-# MANIFESTS_URL=https://github.com/USER/gcp-blueprints.git/packages/management@BRANCH
-
-# Directory where manifests should be fetched to
-MANIFESTS_DIR=./upstream/management
+PACKAGE_URL?=https://github.com/kubeflow/gcp-blueprints.git/packages/management@master
 
 INSTANCE_DIR=./instance
 
+# Directory where packages should be read from. Export these variables, so
+# recursive make commands can get them.
+export PACKAGE_DIR=./upstream/management
+export PACKAGE_CLUSTER_DIR=$(INSTANCE_DIR)/cluster
```

Comment:
For my education, why do we have a variable for instance folder but not for `./upstream` for instance? Will there be multi-instance support?

Reply:
Because the instance folder is just an empty kustomization.yaml that points to the upstream folder.